tests/test_ops/test_scatter_points.py

```diff
@@ -11,7 +11,7 @@ def test_dynamic_scatter():
     feats = torch.rand(
         size=(200000, 3), dtype=torch.float32, device='cuda') * 100 - 50
     coors = torch.randint(
-        low=-1, high=20, size=(200000, 3), dtype=torch.int32, device='cuda')
+        low=0, high=20, size=(200000, 3), dtype=torch.int32, device='cuda')
 
```

Comment:
why should we change this? I personally think the original test that simulates input with invalid coors (have value -1) is actually beneficial. We actually could not guarantee that the output coors of dynamic voxelization are all valid according to the logics here:
https://github.com/open-mmlab/mmcv/blob/a0127e1712cd58b22736d20ed4e5531e1d277f5e/mmcv/ops/csrc/common/cuda/voxelization_cuda_kernel.cuh#L22-L49

Reply:
Sorry for the mistake. I found that some of the unittest in mmdet3d has been deleted here. The op cannot pass the unittest after the recovery of the unittest before this PR. Then I need to recover the unittest to implement the strict testing for the DynamicScatter op.

Reply:
LGTM